uwblueprint/community-fridge-kw
# build: set up Vault config and scripts

## Notion ticket link
https://www.notion.so/uwblueprintexecs/Configure-Vault-791a04d682954265a45f5e50988af0dd



## Implementation description
* Set up Vault configurations for frontend +  root env files and Google application credentials for Firebase.



## Steps to test
1. Follow the documentation here to install  + set up Vault:
https://www.notion.so/uwblueprintexecs/Secret-Management-2d5b59ef0987415e93ec951ce05bf03e
2. Run the following command to retrieve the key/values in Vault
`vault kv get kv/community-fridge-kw`
Check that it contains the key-value pairs for each of these files:
- FRONTEND_ENV_FILE
- ROOT_ENV_FILE
- GOOGLE_APPLICATION_CREDENTIALS
This means that you can properly access the key-value pairs stored in Vault, and now you want to run the following command that will update the three mentioned env/config files.
`vault kv get kv/community-fridge-kw | python update_secret_files.py`
3. Make sure you can still run the application without any errors



## What should reviewers focus on?
* 


## Checklist
